be/src/vec/functions/function_helpers.cpp

```diff
@@ -148,7 +148,7 @@ void validate_argument_type(const IFunction& func, const DataTypes& arguments,
 const ColumnConst* check_and_get_column_const_string_or_fixedstring(const IColumn* column) {
     if (!is_column_const(*column)) return {};
 
-    const ColumnConst* res = assert_cast<const ColumnConst*>(column);
```

Comment:
maybe not need

Reply:
Const property is guaranteed by `is_column_const`